### ISSUE TITLE
Failed to parse source map from 'xxx/node_modules/@reduxjs/toolkit/dist/redux-toolkit.esm.ts'

### ISSUE THREAD

**Issue by tjx666** [reduxjs/redux-toolkit#1704]
Reproduce:

1. clone https://github.com/tjx666/webpack-sourcemap-issue-reproduce.git
2. install dependencies and yarn start

![image](https://user-images.githubusercontent.com/41773861/140647643-37cb77e1-0751-4999-89c7-ae2bda4025c3.png)

**Comment by tjx666** [comment]
Maybe this is an issue with https://github.com/webpack-contrib/source-map-loader

**Comment by chyzwar** [comment]
Problem is because 

redux-toolkit.esm.js.map

```json
{
    "version": 3,
    "sources": [
        "../src/index.ts",
        "../src/createDraftSafeSelector.ts",
        "../src/configureStore.ts",
        "../src/devtoolsExtension.ts",
        "../src/isPlainObject.ts",
        "../src/getDefaultMiddleware.ts",
        "../src/utils.ts",
        "../src/immutableStateInvariantMiddleware.ts",
        "../src/serializableStateInvariantMiddleware.ts",
        "../src/createAction.ts",
        "../src/createReducer.ts",
        "../src/mapBuilders.ts",
        "../src/createSlice.ts",
        "../src/entities/entity_state.ts",
        "../src/entities/state_selectors.ts",
        "../src/entities/state_adapter.ts",
        "../src/entities/utils.ts",
        "../src/entities/unsorted_state_adapter.ts",
        "../src/entities/sorted_state_adapter.ts",
        "../src/entities/create_adapter.ts",
        "../src/nanoid.ts",
        "../src/createAsyncThunk.ts",
        "../src/tsHelpers.ts",
        "../src/matchers.ts",
        "redux-toolkit.esm.ts"
    ]
}
```

And  *redux-toolkit.esm.ts* do not exist in dist folder.

**Comment by phryneas** [comment]
Is that still the case with the new 1.7.0 beta?

**Comment by HolyZheng** [comment]
> Is that still the case with the new 1.7.0 beta?

yes~, I have tried 1.7.0 beta.1, but still have this problem

**Comment by futurechallenger** [comment]
This will work:
```js
"@reduxjs/toolkit": "^1.3.0-beta.1",
```

**Comment by markerikson** [comment]
@futurechallenger referring back to a very old version of RTK really isn't helpful information here :(

**Comment by futurechallenger** [comment]
> 

Yeah, find that just now... tried 1.7.0-beta-1, got the same error.

**Comment by markerikson** [comment]
Should now hopefully be fixed in https://github.com/reduxjs/redux-toolkit/releases/tag/v1.7.0-rc.0 . Can someone check and confirm?

**Comment by futurechallenger** [comment]
![image](https://user-images.githubusercontent.com/383584/143729144-5865a3e0-acb0-4ecb-9b19-aeae599c9466.png)
@markerikson it works

**Comment by markerikson** [comment]
Fixed in 1.7.
